src/driver/driver.cpp

```diff
@@ -162,15 +154,24 @@ void EvolutionDriver::SetGlobalTimeStep() {
 
 void EvolutionDriver::OutputCycleDiagnostics() {
   const int dt_precision = std::numeric_limits<Real>::max_digits10 - 1;
-  const int ratio_precision = 3;
   if (tm.ncycle_out != 0) {
     if (tm.ncycle % tm.ncycle_out == 0) {
       if (Globals::my_rank == 0) {
+        std::uint64_t zonecycles =
+            (pmesh->mbcnt - mbcnt_prev) *
```

Comment:
Sorry, maybe my brain isn't working so good right now - why do we only multiply by the number of new blocks since the last output? Maybe I'm not understanding what's going no here.

Reply:
This allows to calculate the performance of the last cycle (how many blocks updated in the last cycle with how many cells over what time).

This could be interesting to also see how load balancing and writing outputs affects the performance (now that I wrote it we could even output the dwtime to track this independently of the blocks, but that may be for later when as it'd require additional changes to really become informative, such as including time to write output, load balanace, etc).